client/dashboard/task-list/tasks/payments/index.js

```diff
@@ -131,7 +135,7 @@ class Payments extends Component {
 								'and one-touch checkout with Apple Pay.',
 							'woocommerce-admin'
 						) }
-						{ this.renderWooCommerceServicesStripeConnect( formData ) }
+						{ this.renderWooCommerceServicesStripeConnect( this.formData ) }
```

Comment:
Looks like this no longer needs to be passed `this.formData`.

Reply:
Thanks for catching these. Fixed in https://github.com/woocommerce/woocommerce-admin/pull/2858/commits/c48668bbb993ab28b66e819b6f4770f40817d1d2.